tests/core/test_views_swagger.py

```diff
@@ -37,3 +37,25 @@ def test_extensions(self):
             with mock.patch('pkg_resources.resource_filename', lambda pkg, f: path):
                 response = self.app.get('/swagger.json')
                 self.assertEquals(response.json['swagger'], '3.0')
+
+    def test_default_security_extensions(self):
+        path = tempfile.mktemp(suffix='.yaml')
+        content = {
+            'securityDefinitions': {
+                'fxa': {
+                    'type': 'oauth2',
+                    'authorizationUrl': 'https://oauth-stable.dev.lcip.org',
+                    'flow': 'implicit',
+                    'scopes': [{'kinto': 'Basic scope'}]
+                }
+            }
+        }
```

Comment:
This test case (in comparison to the last one) checks if it's possible to change the default security (which means what the resources require as authentication) by only updating only `securityOptions` on an extension. It matches what happens on [this part](https://github.com/gabisurita/kinto/blob/775b67d6c78ae011286946f53bd99af0602b1e57/kinto/core/views/swagger.py#L67). I didn't test for type in this case because it should be the same for `test_extensions`.